feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -781,6 +788,13 @@ class BillingWrapper(
 
         return BillingFlowParams.newBuilder()
             .setProductDetailsParamsList(listOf(productDetailsParamsList))
+            .apply {
+                // only setObfuscatedAccountId for non-upgrade/downgrades until google issue is fixed:
+                // https://issuetracker.google.com/issues/155005449
```

Comment:
I don't have access to this issue, so I'm actually not sure what the issue was... I'm leaving the logic as it was just in case though